soh/soh/Enhancements/crowd-control/CrowdControl.cpp

```diff
@@ -333,13 +282,13 @@ CrowdControl::Effect* CrowdControl::ParseMessage(char payload[512]) {
             effect->category = kEffectCatDamageTaken;
             effect->timeRemaining = 30000;
             effect->giEffect = new GameInteractionEffect::ModifyDefenseModifier();
-            effect->giEffect->parameters[0] = 2;
+            dynamic_cast<ParameterizedGameInteractionEffect*>(effect->giEffect)->parameters[0] = 2;
```

Comment:
generally good practice to throw a `!= nullptr` in there somewhere when dynamic casting, can be a follow up PR if you want to make a code cleanup issue for it